lib/tasks/attempts.rake

```diff
@@ -13,29 +13,24 @@ namespace :attempts do
         "Bearer #{IdentityConfig.store.irs_attempt_api_csp_id} #{auth_token}"
     end
 
-    encrypted_data = Base64.strict_decode64(resp.body)
-    iv = Base64.strict_decode64(resp.headers['x-payload-iv'])
-    encrypted_key = Base64.strict_decode64(resp.headers['x-payload-key'])
-    private_key = OpenSSL::PKey::RSA.new(File.read(private_key_path))
-    key = private_key.private_decrypt(encrypted_key)
-    decrypted = IrsAttemptsApi::EnvelopeEncryptor.decrypt(
-      encrypted_data: encrypted_data, key: key, iv: iv,
-    )
+    puts "Response Body:"
+    puts resp.body
+
+    events = resp.body.split("\r\n")
 
-    events = JSON.parse(decrypted)
+    private_key = OpenSSL::PKey::RSA.new(File.read(private_key_path))
 
     if File.exist?(private_key_path)
       puts events.any? ? 'Decrypted events:' : 'No events returned.'
 
-      events.each do |_jti, jwes|
-        jwes.each do |_key_id, jwe|
-          begin
-            pp JSON.parse(JWE.decrypt(jwe, private_key))
-          rescue
-            puts 'Failed to parse/decrypt event!'
-          end
-          puts "\n"
+      events.each do |jwe|
+        begin
+          puts JSON.parse(JWE.decrypt(jwe, private_key))
+        rescue => e
+          puts 'Failed to parse/decrypt event!'
+          puts e.inspect
```

Comment:
Clean this up, but I added this when we were failing on `private_key` being undefined and couldn't see why.

We eliminate some of the inner complexity by not doing nested JSON stuff.

Reply:
Done!